### PR TITLE
hello-openshift: fix base images

### PR DESCRIPTION
**Pull request by yselkowitz**
/cc @sgreene570

### CODE REVIEW

File: images/hello-openshift/Dockerfile.rhel [hunk 1]
```diff
@@ -1,11 +1,9 @@
-#FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-openshift-4.6 AS builder
-FROM registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:rhel_8_golang_1.14 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-openshift-4.6 AS builder
```
Comment (sgreene570) [inline images/hello-openshift/Dockerfile.rhel:1]: should this image be `rhel-8-golang-1.15-openshift-4.6` or is that the new default in this case?

Comment (yselkowitz) [reply images/hello-openshift/Dockerfile.rhel]: you're right, a bit preoccupied atm.

Comment (sgreene570) [reply images/hello-openshift/Dockerfile.rhel]: np! Thanks for making this PR.

**Pull request merged**